graphql-server/go-server/main.go

```diff
@@ -85,6 +86,12 @@ func main() {
 		http.Handle("/bff", srv)
 	}
 
+	http.HandleFunc("/minio/get_chunks", minio.GetSuccessChunks)
```

Comment:
We need to do permission check in namespaced scope.

Reply:
TODO